lib/manageiq/schema/schema_dumper.rb

```diff
@@ -52,8 +52,7 @@ def miq_metric_table_constraints(stream)
           conditions        = [primary_condition, "realtime", child_table_num]
 
           stream.puts "  add_miq_metric_table_inheritance #{child_table}, " \
-                          "#{inherit_from.inspect}, "                       \
-                          ":conditions => #{conditions.inspect}"
+                          "#{inherit_from.inspect} "
```

Comment:
You can remove a number of the local variables now such as conditions, primary_condition, and child_table_num.